src/saving_files.jl

```diff
@@ -24,7 +24,7 @@ end
 
 ## Keywords
 * `suffix = "jld2", prefix = default_prefix(config)` : Used in [`savename`](@ref).
-* `tag::Bool = get(ENV, "DRWATSON_TAG", istaggable(suffix))` : Save the file
+* `tag::Bool = readenv("DRWATSON_TAG", istaggable(suffix))` : Save the file
```

Comment:
I now realize the problem here. `readenv` is neither an exported name, nor does it have a docstring. How does it help a user to see `readenv` here? They know nothing about it, nor they can learn anything about it. I don't think this is useful for a documentation string. 

Not sure what the best way is, but I definitely prefer the original way. Even though it is not "correct", it does convey what is happening.

Reply:
I agree that it's not the ideal solution right now. Though reading the docstring was a red herring to me, and that caused me to do this PR. I wasn't using this functionality but had been working with `ENV` in another project. Leaving it as it was before will lead potentially many other users to read the code and "try to fix this" even though it has already been fixed.

I would instead just add a docstring to `readenv` but I wouldn't export it.

Reply:
okay, let's add a docstring to `readenv`, and also replace it with `DrWatson.readenv` in the docs!

Reply:
> also replace it with DrWatson.readenv in the docs!

Had the same idea. 😄